### PR TITLE
Add information about publication: Energy storage in Central Asia

### PR DESCRIPTION
**Pull request by behnam-zakeri**
This PR adds information on one of publications of MESSAGEix for the list in the "usage" page.

## How to review

Please check out this branch, build the documentation on your machine, and check if the information in the page "[Usage](https://docs.messageix.org/en/stable/usage.html)" about [this publication](https://www.sciencedirect.com/science/article/pii/S2352152X2200603X) reads correctly.

## PR checklist
- [x] Continuous integration checks all ✅
- [x] Tests not needed ✅
- [x] Add, expand, or update documentation.
- [x] Release notes not needed.

### CODE REVIEW

**Review by glatterf42 — CHANGES_REQUESTED** [review]
I applied black so that the lint check passes. In that process, I also noticed that that we should move the coverage configuration to pyproject.toml as explained [here](https://iiasa-ece.slack.com/archives/CD0GBHHA4/p1674121353182649).
Regarding the documentation, I think it looks good. However, I'd like to see a small change to the phrasing of 'Usage' to make it consistent with the other projects: please use the imperative form, e.g. 'Model the energy-water system of Central Asia with 12 sub-annual time slices and analyze the role of energy and water storage solutions.'
Once that is done (and the checks have passed again), I'll merge this PR.

File: doc/usage.rst [hunk 1]
```diff
@@ -23,6 +23,21 @@ Publications
...
+-----------------------------------------------------------------------------------
+.. figure:: _static/usage_figures/10.1016-j.est.2022.104587.JPG
+   :width: 250px
+   :align: right
+
+:cite:ct:`Zakeri2022`
+
+- **Spatial**: Regional (Central Asia), multi-country including Kazakhstan, Kyrgyzstan, Tajikistan, Turkmenistan, and Uzbekistan
+- **Keywords**: Energy storage, seasonal pumped hydropower storage, water management, renewable energy systems, energy policy, electricity storage, energy model
+- **Usage**: An energy-water system model of Central Asia with 12 sub-annual time slices for modeling and analyzing the role of energy and water storage solutions.
```
Comment (glatterf42) [inline doc/usage.rst:36]:
```suggestion
- **Usage**: Model the energy-water system of Central Asia with 12 sub-annual time slices and analyze the role of energy and water storage solutions.
```

**Review by glatterf42 — COMMENTED** [review]
Add the suggestion so that it may be applied directly.

**Comment by behnam-zakeri** [comment]
Thanks @glatterf42 for reviewing this. Very good suggestion for the text, I applied that. The lint errors you mentioned earlier are not related to this PR. This PR originally didn't touch the code, and was only an update to the documentation. So, I assume there has been changes in the version Black that caused new lint errors, or maybe we needed to rebase this PR.

**Review by glatterf42 — APPROVED** [review]

**Pull request merged**